smsdk/client_v0.py

```diff
@@ -117,12 +119,26 @@ def __init__(self, tenant, site_domain="sightmachine.io"):
         :type site_domain: :class:`string`
         """
 
-        self.tenant = tenant
-
-        # Handle internal configuration
-        self.config = {}
-        self.config["protocol"] = "https"
-        self.config["site.domain"] = site_domain
+        if "://" in tenant:  # Full URL provided eg. "https://demo.sightmachine.io"
+            protocol, domain = tenant.split("://")[0], tenant.split("://")[1]
```

Comment:
If this is suppose to be a valid URI, can use `urllib.parse.parse_url`.
```python
import urllib.parse
parsed = urllib.parse.urlparse(tenant)
protocol, domain = parsed.schema, parsed.netloc
```
The alternative is the partition command.
```python
protocol, _, domain = tenant.partition("://")
```

Reply:
Done,